component/loki/source/podlogs/reconciler.go

```diff
@@ -101,6 +126,31 @@ func (r *reconciler) Reconcile(ctx context.Context, cli client.Client) error {
 	return nil
 }
 
+func distributeTargets(c cluster.Cluster, targets []*kubetail.Target) []*kubetail.Target {
+	if c == nil {
+		return targets
+	}
+
+	res := make([]*kubetail.Target, 0, (len(targets)+1)/len(c.Peers()))
+
+	// TODO(@tpaschalis): Make sure OpReadWrite is the correct operation;
+	// eg. this determines how clustering behaves when nodes are shutting down.
```

Comment:
OpReadWrite _is_ the correct operation here, I think we can remove this TODO comment